hack/ci/e2e-k8s.sh

```diff
@@ -115,7 +115,9 @@ create_cluster() {
       apiServer_extra_args="${apiServer_extra_args}
       \"logging-format\": \"${CLUSTER_LOG_FORMAT}\""
   fi
-  kubelet_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
+  kubelet_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\"
+      \"container-log-max-files\": \"10\"
+      \"container-log-max-size\": \"100Mi\""
```

Comment:
I'm concerned whenever we have to do CI-only configuration @aojea, because it means we're no longer actually testing the defaults, do we think this is because of the verbosity on the components?

Reply:
compared with an e2e gce log the sizes are similar https://gcsweb.k8s.io/gcs/kubernetes-ci-logs/pr-logs/pull/128474/pull-kubernetes-e2e-gce/1852746024750157824/artifacts/e2e-44626956c7-674b9-master/

the difference is that those components logs to the filesystem, with the gorunner thingy

Reply:
> the difference is that those components logs to the filesystem, with the gorunner thingy, in kind everything logs to the pods and is rotated by the kubelet IIRC

This seems like a gap to revisit later, ideally we want to make sure component logs have robust retention, but maybe not random pods.